bsm_helper/include/bsm_helper/bsm_helper.h

```diff
@@ -0,0 +1,29 @@
+#include <string>
+#include <vector>
+#include <exception>
+#include <ros/ros.h>
+#include <mutex>
+#include <memory>
+#include <cav_msgs/BSMCoreData.h>
+#include <boost/optional.hpp>
+
+namespace BSMHelper {
+
+    class BSMHelper
+    {
+        public:
+
+            /**
+             * @brief Converts bsm id from uint8_t vector to hex string
+             * @param id The incoming bsm id
```

Comment:
can you add a bit of description about the encoding (how the input array needs to be structured)